catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/SceneListFragment.java

```diff
@@ -174,8 +176,9 @@ protected void deleteItems(List<Scene> selectedItems) {
 			createEmptySceneWithDefaultName();
 		}
 
-		if (ProjectManager.getInstance().getCurrentProject().getSceneList().size() < 2) {
-			switchToSpriteListFragment();
+		Project currentProject = ProjectManager.getInstance().getCurrentProject();
+		if (currentProject.getSceneList().size() < 2) {
+			switchToSpriteListFragment(currentProject);
```

Comment:
You added the parameter so that whenever a scene was removed you make sure that the `currentlyEditedScene` is updated correctly, which absolutely makes sense. However, how about you set it directly instead of going the extra way around it by using `setCurrentProject(..)`, as it would make it easier to understand what you do and why it is necessary.